index.html

```diff
@@ -0,0 +1,450 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Youtube Page</title>
+	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
+	 crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="./assets/stylesheet/style.css">
+
+</head>
+
+<body>
+	<header>
+		<div class="nav-bar">
+			<section class="header-left">
+				<i class="fas fa-bars"></i>
+				<img class="logo" src="./assets/media/images/youtube.png" alt="logo">
+			</section>
+			<section class="header-center">
+				<input type="text" class="search-field" placeholder="Search">
+				<button class="search-button">
+					<i class="fas fa-search"></i>
+				</button>
+			</section>
+			<section class="header-right">
+				<i class="fas fa-video"></i>
+				<i class="fas fa-th"></i>
+				<i class="fas fa-comment"></i>
+				<i class="fas fa-bell"></i>
+				<img class="user" src="./assets/media/images/user.png" alt="user">
+			</section>
+		</div>
+
+	</header>
+	<main>
+		<section class="main-video">
+
+			<iframe width="560" height="450" src="https://www.youtube.com/embed/CvJG4sQhzsw" frameborder="0" allow="autoplay; encrypted-media"
+			 allowfullscreen></iframe>
+			<p>The fastest way to become a software developer</p>
+			<p class="comments">44,000 views
+				<span class="rightComments">
+					<span class="likesDislikes">
+						<i class="fas fa-thumbs-up"> </i> 400
+						<i class="fas fa-thumbs-down"> </i>9
+					</span>
+					<i class="fas fa-share"> </i>SHARE
+					<i class="fas fa-plus-square"> </i>
+					<i class="fas fa-ellipsis-h"> </i>
+				</span>
+			</p>
+			<hr>
+
+			<div class="videoDescription">
+				<div class="author-avatar">
+					<img src="./assets/media/images/user.png" alt="profile-picture">
+				</div>
+
+
+				<div class="descTitle">
+
+
+					<div class="by-line">
+						<p>TheHappieCat</p>
+						<p class="date-published">Published on 2 Mar 2016</p>
+					</div>
+					<div class="subscribe-button">
+						<button>SUBSCRIBE
+							<span> 412K </span>
+						</button>
+					</div>
+					<div class="clear-float">
```

Comment:
Having one of this between every element of the column feels like a "hack". I don't think generating an extra div for every column element is the best way to go

Reply:
We were using floats and needed to clear it. That was the best we could come up with at the time. What would you advise?

Reply:
I mean at this point we can let it go. What I'm trying to say it's that it's not usual to see A LOT of empty <div> in a webpage just to make a design (and in this case, a list) look nice.